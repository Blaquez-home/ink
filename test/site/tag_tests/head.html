<meta name='generator' content='Octopress 1.0.0.alpha.19'>
Testing head tag
Testing that content_for is additive

