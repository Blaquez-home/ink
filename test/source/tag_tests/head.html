---
layout: default
---
{% head %}
Testing head tag
{% endhead %}

{% head %}
Testing that content_for is additive
{% endhead %}
