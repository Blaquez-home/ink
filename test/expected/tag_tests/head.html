<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name='generator' content='Octopress 1.0.0.alpha.15'>
Testing head tagTesting that content_for is additive
  </head>
  <body>
    
    
    
  </body>
</html>
